lib/index.js

```diff
@@ -331,22 +341,31 @@ function _validateReceiveHeader(header, value) {
       header + ' was not in the request', 'SyntaxError');
   }
 }
+function addValue({result, validate, key, values, label}) {
+  let value = values[key];
+  validate(key, value);
+  if(Array.isArray(value)) {
+    value = value.join(', ');
+  }
+  result.push(`${label || key}: ${value}`);
+}
 
-function _signingString(
-  {includeHeaders, headers, method, url, host, path, validate}) {
+function _signingString({includeHeaders, headers, method, url,
+  host, path, requestOptions, validate}) {
   const result = [];
   for(const h of includeHeaders) {
     if(h === '(request-target)') {
       result.push(`(request-target): ${method.toLowerCase()} ${path || url}`);
     } else if(h === 'host' && host) {
       result.push(`host: ${host}`);
     } else {
-      let value = headers[h];
-      validate(h, value);
-      if(Array.isArray(value)) {
-        value = value.join(', ');
+      const pseudoHeader = PSEUDO_HEADERS[h.trim()];
+      if(pseudoHeader) {
+        addValue({result, validate, key: pseudoHeader,
```

Comment:
Fix unusual line breaks

Reply:
style guide?

Reply:
```js
addValue({
  result, validate, key: pseudoHeader,
  values: requestOptions, label: h
});
```

Reply:
Addressed in https://github.com/digitalbazaar/http-signature-header/pull/12/commits/1f18f7e132e46beba7bcca9f7eeca115e1bb90d4